src/ctap/crypto_wrapper.rs

```diff
@@ -173,17 +193,18 @@ impl PrivateKey {
     /// Writes the key bytes.
     pub fn to_bytes(&self) -> Vec<u8> {
         match self {
-            PrivateKey::Ecdsa(ecdsa_key) => {
-                let mut key_bytes = vec![0u8; 32];
-                ecdsa_key.to_bytes(array_mut_ref!(key_bytes, 0, 32));
-                key_bytes
-            }
+            PrivateKey::Ecdsa(ecdsa_seed) => ecdsa_seed.to_vec(),
```

Comment:
Does this affect the stored bytes, and is it a breaking change? Or is every transformation afterwards the identity?

Reply:
For both resident keys and server-side?

Reply:
Yes.